.github/workflows/pr_and_main.yml

```diff
@@ -85,9 +85,7 @@ jobs:
       - run: ctest --preset test
       - run: python -m pytest -v tests
 
-      - run: |
-          pip install sphinx-autodoc-typehints>=1.17.0
-          cmake --build --preset docs
+      - run: cmake --build --preset docs
```

Comment:
So all systems install the docs dependencies (as before), but only one build uses it (this was also the case before, just asking to clarify)?